### PR TITLE
Incorporating Rebranding Changes for Mobile Cards, iOS SDK & Android SDK

### PR DESCRIPTION
**Pull request by sandeep-tds**
Made the following changes:

replace "software-product-function>sap-cloud-platform-mobile-services"
with "products>sap-mobile-services"

replace "products>sap-cloud-platform"
with "products>sap-business-technology-platform"

replace "products>sap-cloud-platform-for-the-cloud-foundry-environment"
with "products>sap-btp-cloud-foundry-environment"

replace "SAP Cloud Platform SDK for iOS"
with "SAP BTP SDK for iOS"

replace "SAP BTP SDK for iOS Assistant"
with "SAP BTP SDK Assistant for iOS"

replace "SAP Cloud Platform SDK for Android"
with "SAP BTP SDK for Android"

replace "SAP Cloud Platform SDK for Android Wizard"
with "SAP BTP SDK Wizard for Android"

replace "SAP Cloud Platform Mobile Services"
with "SAP Mobile Services"

replace "SAP Cloud Platform Mobile Service"
with "SAP Mobile Services"

replace "SAP Cloud Platform"
with "SAP BTP"

replace "SCP"
with "SAP BTP"

replace "SAP CP"
with "SAP BTP"

replace "SAP Mobile Services"
with "SAP Mobile Services"

### CODE REVIEW

File: tutorials/cp-sdk-android-flows-custom-step/cp-sdk-android-flows-custom-step.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 title: Create a Custom Step in a Flow
-description: Create a custom step in a flow which will read JSON data from the SAP Cloud Platform Mobile Service's JSON storage feature.
+description: Create a custom step in a flow which will read JSON data from the SAP Mobile Services's JSON storage feature.
 auto_validation: true
 time: 20
-tags: [ tutorial>beginner, products>sap-cloud-platform, operating-system>android, topic>mobile]
-primary_tag: products>sap-cloud-platform-sdk-for-android
+tags: [ tutorial>beginner, products>sap-business-technology-platform, operating-system>android, topic>mobile]
+primary_tag: products>sap-business-technology-platform-sdk-for-android
```
Comment (MichaelCzcz) [inline tutorials/cp-sdk-android-flows-custom-step/cp-sdk-android-flows-custom-step.md:7]: products>android-sdk-for-sap-btp

File: tutorials/cp-sdk-android-flows-restore/cp-sdk-android-flows-restore.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
 title: Add a Restore Flow
-description: Add a restore flow that executes when the app is reopened.  The restore flow will present the passcode screen, unlock the secure stores and update the client policy specified in the SAP Cloud Platform Mobile Services.
+description: Add a restore flow that executes when the app is reopened.  The restore flow will present the passcode screen, unlock the secure stores and update the client policy specified in the SAP Mobile Services.
 auto_validation: true
 time: 20
-tags: [ tutorial>beginner, products>sap-cloud-platform, operating-system>android, topic>mobile]
-primary_tag: products>sap-cloud-platform-sdk-for-android
+tags: [ tutorial>beginner, products>sap-business-technology-platform, operating-system>android, topic>mobile]
+primary_tag: products>sap-business-technology-platform-sdk-for-android
```
Comment (MichaelCzcz) [inline tutorials/cp-sdk-android-flows-restore/cp-sdk-android-flows-restore.md:7]: products>android-sdk-for-sap-btp
--

File: tutorials/cp-sdk-android-flows-stores/cp-sdk-android-flows-stores.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ---
 title: Explore the Secure Stores Created by the Onboarding Flow
-description: Explore the secure stores created by the onboarding flow and examine the persisted data such as the SAP Cloud Platform Mobile Services host, credentials and client policy.
+description: Explore the secure stores created by the onboarding flow and examine the persisted data such as the SAP Mobile Services host, credentials and client policy.
 auto_validation: true
 time: 10
-tags: [ tutorial>beginner, products>sap-cloud-platform, operating-system>android, topic>mobile ]
-primary_tag: products>sap-cloud-platform-sdk-for-android
+tags: [ tutorial>beginner, products>sap-business-technology-platform, operating-system>android, topic>mobile ]
+primary_tag: products>sap-business-technology-platform-sdk-for-android
```
Comment (MichaelCzcz) [inline tutorials/cp-sdk-android-flows-stores/cp-sdk-android-flows-stores.md:7]: products>android-sdk-for-sap-btp
--

File: tutorials/cp-sdk-android-wizard-app-customize/cp-sdk-android-wizard-app-customize.md [hunk 4]
```diff
@@ -3,14 +3,14 @@ author_name: Bruce Meng
 author_profile: https://github.com/flyingfish162
 title: Customize the Wizard Generated Application
 description: Customize a wizard-generated application and learn how to use Fiori object cells, search UI, and the collection view.
-primary_tag: products>sap-cloud-platform-sdk-for-android
+primary_tag: products>sap-business-technology-platform-sdk-for-android
```
Comment (MichaelCzcz) [inline tutorials/cp-sdk-android-wizard-app-customize/cp-sdk-android-wizard-app-customize.md:6]: products>android-sdk-for-sap-btp

File: tutorials/cp-sdk-android-wizard-app-customize/cp-sdk-android-wizard-app-customize.md [hunk 5]
```diff
@@ -3,14 +3,14 @@ author_name: Bruce Meng
...
 auto_validation: true
-tags: [  tutorial>beginner, operating-system>android, topic>mobile, topic>wizard, products>sap-cloud-platform-sdk-for-android, products>sap-cloud-platform ]
+tags: [  tutorial>beginner, operating-system>android, topic>mobile, topic>wizard, products>sap-business-technology-platform-sdk-for-android, products>sap-business-technology-platform ]
```
Comment (MichaelCzcz) [inline tutorials/cp-sdk-android-wizard-app-customize/cp-sdk-android-wizard-app-customize.md:8]: products>android-sdk-for-sap-btp

File: tutorials/cp-sdk-android-wizard-app-logging/cp-sdk-android-wizard-app-logging.md [hunk 6]
```diff
@@ -3,20 +3,20 @@ author_name: Bruce Meng
 author_profile: https://github.com/flyingfish162
 title: See How Logging Can Be Used in Your Android Application
 description: Explore how the logging component can help make deployed applications more supportable.
-primary_tag: products>sap-cloud-platform-sdk-for-android
+primary_tag: products>sap-business-technology-platform-sdk-for-android
 auto_validation: true
-tags: [  tutorial>beginner, operating-system>android, topic>mobile, products>sap-cloud-platform-sdk-for-android, products>sap-cloud-platform ]
+tags: [  tutorial>beginner, operating-system>android, topic>mobile, products>sap-business-technology-platform-sdk-for-android, products>sap-business-technology-platform ]
```
Comment (MichaelCzcz) [inline tutorials/cp-sdk-android-wizard-app-logging/cp-sdk-android-wizard-app-logging.md:8]: products>android-sdk-for-sap-btp

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please make sure to use the correct tags. You can't simply replace the part that says "cloud" with "business-technology".
I have commented on some instances where you did this, but there are many more.
I will forward Daniel's email regarding BTP re-branding to you.

**Comment by sandeep-tds** [comment]
Hi Michael, 

Thank you for raising the issue. I will correct these tags and submit a new commit.

**Comment by sandeep-tds** [comment]
Hi @thecodester, @MichaelCzcz 

@thecodester - I have made the changes we discussed. Can you please review the Pull Request and merge the changes? 
 
Thank you so much for your support.

**Comment by sandeep-tds** [comment]
@thecodester  Thank you so much for the swift action. 

FYI @KevinMuessig  @flyingfish162 @FengHaoyue  - I've updated the tags for our tutorials and made all the rebranding changes. I am working on getting the community tag updated to our latest names. Once done, I will create a new PR. Thank you all so much for your support.

**Pull request merged**